src/IBM.WatsonDeveloperCloud/Sockets/AWatsonWSClient.cs

```diff
@@ -0,0 +1,153 @@
+﻿/**
```

Comment:
Ok German! I changed for AWebSocketClient, the abstract class, and WebSocketClient, the concrete class.